modules/mono/glue/GodotSharp/GodotSharp/Core/Bridge/ScriptManagerBridge.cs

```diff
@@ -131,6 +131,9 @@ internal static void FrameCallback()
                 // Performance is not critical here as this will be replaced with source generators.
                 Type scriptType = _scriptTypeBiMap.GetScriptType(scriptPtr);
 
+                if (scriptType.IsAbstract)
+                    return godot_bool.False;
```

Comment:
Is this really needed? I don't think this method should be called for non-instantiable classes so this should never be reachable.

And if it's called for an abstract class, wouldn't it be better to let it throw an exception when it tries to instantiate the abstract class?

Reply:
I tried to do a little bit of searching (which consists just of skimming through the function calls in `csharp_script.cpp`, not sure how much it's called elsewhere) and it doesn't seem like there are any safety checks to make sure that a class is instantiable.

I agree though that an exception may be better in this case, but I would rather throw it right after the check for the extra clarity instead of waiting for the constructor to be called. Maybe something like this?
```suggestion
                if (scriptType.IsAbstract)
                    throw new ArgumentException(
                        $"Cannot create script instance. The class '{scriptType.FullName}' is abstract.");
```

Reply:
Alright, I think that sounds good.